README.md

```diff
@@ -32,18 +32,34 @@ Or install it yourself as:
 ## Usage
 
 ```ruby
-Identifiers::DOI.extract('example: 10.123/abcd.efghi')
-# => ["10.123/abcd.efghi"]
+Identifiers::DOI.extract('example: 10.1234/5678.abc')
```

Comment:
I wonder if this example should also show normalisation ?

Reply:
Sure - I can update it. The only reason I changed this DOI was because the one we had before was invalid and would return an empty array. I'll change the text so that it has `10.1234/5678.ABC` instead.